File: ESC.Resturant.Data/Entities/Branch.cs
using ESA.Data;
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.Linq;
using System.Threading.Tasks;

namespace ESC.Resturant.Data.Entities
{
    public partial class Branch
    {
        [key]
        public int Id { get; set; }
        public string NameAr { get; set; }
        public string NameEn { get; set; }
        public bool IsDelete { get; set; }
        public int? CompanyInfoId { get; set; }
        public virtual CompanyInfo CompanyInfo { get; set; }
        public virtual ICollection<Bill> Bills { get; set; }
        public virtual ICollection<ApplicationUser> ApplicationUsers { get; set; }

    }
}

File: ESAResturant.NEW.web/ClientApp/src/app/admin/setting/Models/Branch.ts
import { Bill } from "../../bill-invoice/Models/Bill";
import { CompanyInfo } from "../../../users/Models/CompanyInfo";

export class Branch {

  constructor() {

    this.id = 0;
    this.nameAr = '';
    this.nameEn = '';
    this.isDelete = false;
    this.companyInfoId = 0;

    this.companyInfo = new CompanyInfo();
  }

  id: number;
  nameAr: string;
  nameEn: string;
  isDelete: boolean;
  companyInfoId: number;
  companyInfo: CompanyInfo;
}
